src/osd/OSD.h

```diff
@@ -1934,15 +1934,15 @@ class OSD : public Dispatcher,
 	}
       }
     }
-    bool _enqueue(PG *pg) {
+    bool _enqueue(PG *pg) override {
```

Comment:
> I don't have much experience with noexcept

neither do i =)

i think the actual questions are:

1. we need to mark the overrided method in the parent class with `noexcept`, if we want to have it in the child class.
1. shall we `terminate()` at seeing an exception we do not handle?